linkatos/activities.py

```diff
@@ -9,71 +9,69 @@ def is_empty(events):
     return ((events is None) or (len(events) == 0))
 
 
-def is_url(url_cache):
-    return url_cache is not None
+def is_url(element):
+    return element is not None
 
 
 def is_not_from_bot(bot_id, user_id):
     return not bot_id == user_id
 
 
-def is_empty_list(list):
-    return len(list) == 0
+def is_empty_list(ls):
+    return len(ls) == 0
```

Comment:
I noticed `list` was a key word and thought it might bring issues later on.
I'll move back to it if you think it's unlikely to be problematic.